client/plasmacli/eth/deposit.go

```diff
@@ -0,0 +1,61 @@
+package eth
+
+import (
+	"fmt"
+	ks "github.com/FourthState/plasma-mvp-sidechain/client/keystore"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"math/big"
+	"strconv"
+)
+
+func init() {
+	ethCmd.AddCommand(depositCmd)
+	depositCmd.Flags().StringP(gasLimitF, "g", "21000", "gas limit for ethereum transaction")
+	viper.BindPFlags(depositCmd.Flags())
+}
+
+var depositCmd = &cobra.Command{
+	Use:   "deposit <amount> <account>",
```

Comment:
Are we allowing people to specify account name here or do they have to give hex address? Ideally both options allowed

Reply:
Actually I think that functionality should go in a separate mini-PR